demos/single_leg.py

```diff
@@ -16,7 +16,7 @@
 try:
     while True:
         leg.acquire_sensors()
-        des_pos = leg.get_slider_positions()
+        des_pos = leg.get_slider_positions() - 0.5
```

Comment:
@jviereck is this reasonable?

Reply:
Note sure.

@avadesh02 , why did you change this?